eclipse-collections-api/src/main/java/org/eclipse/collections/api/bag/MultiReaderBag.java

```diff
@@ -41,6 +41,20 @@ default MultiReaderBag<T> without(T element)
         return this;
     }
 
+    @Override
+    default MultiReaderBag<T> withOccurrences(T element, int occurrences)
+    {
+        this.addOccurrences(element, occurrences);
```

Comment:
Just to confirm, we use this pattern for Multi reader collections correct? The return this is why I am asking

Reply:
Yes, the `addOccurrences` method will take a lock, so I didn't introduce the lock in this override.